specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -3160,6 +3160,67 @@
           "type": "object"
         }
       }
+    },
+    "AzureFunctionActivityMethod": {
+      "description": "The list of HTTP methods supported by a AzureFunctionActivity.",
+      "type": "string",
+      "enum": [
+        "GET",
```

Comment:
Make sure that we don't have other types in our swagger that list same HTTP methods. Web activity might need this too, unless it doesn't support the same methods. If they are the same, change the name of this type to more generic - e.g. HttpMethod - then change in both activities to use it.

Reply:
Its different for Azure Function and Web Acitivity. But Web activity and WebHook have the same so ill make that change in the other PR